src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -955,6 +955,9 @@ export function submitAppointmentOrRequest(history) {
     const additionalEventData = {
       'health-TypeOfCare': typeOfCare,
       'health-ReasonForAppointment': data?.reasonForAppointment,
+      'vaos-number-of-preferred-providers': data.ccEnabledSystems
```

Comment:
I think this is the wrong variable to check. This is how many VA systems a user could route CC requests to. But we want to check if there's a provider in `data.communityCareProvider`. And actually, I think you could just use `data.hasCommunityCareProvider`: `data.hasCommunityCareProvider ? 1 : 0`. You can't enter more than one now.